crawl-ref/source/spl-transloc.cc

```diff
@@ -1606,3 +1607,46 @@ void attract_monsters()
         mons_relocated(mi);
     }
 }
+
+bool word_of_chaos(int pow)
+{
+    vector<monster *> targets;
+    for (monster_near_iterator mi(you.pos(), LOS_NO_TRANS); mi; ++mi) 
+    {
+        if (!mons_is_tentacle_or_tentacle_segment(mi->type))
+            targets.push_back(*mi);
+    }
```

Comment:
Is it intentional that this targets friendlies? (Dg interaction danger zone enter here)

Reply:
Fixed, thanks.